pkg/controller/sparkapplication/controller.go

```diff
@@ -500,21 +532,33 @@ func (c *Controller) processSingleExecutorStateUpdate(update *executorStateUpdat
 
 func (c *Controller) updateSparkApplicationStatusWithRetries(
 	original *v1alpha1.SparkApplication,
-	updateFunc func(*v1alpha1.SparkApplicationStatus)) *v1alpha1.SparkApplication {
-	toUpdate := original.DeepCopy()
+	updateFunc func(*v1alpha1.SparkApplicationStatus),
+) *v1alpha1.SparkApplication {
```

Comment:
You can revert this line. It doesn't need to be in a new line.

Reply:
done